core/src/main/java/com/opensymphony/xwork2/security/DefaultExcludedPatternsChecker.java

```diff
@@ -66,9 +76,14 @@ public void setAdditionalExcludePatterns(String excludePatterns) {
     @Inject(StrutsConstants.STRUTS_ENABLE_DYNAMIC_METHOD_INVOCATION)
     public void setDynamicMethodInvocation(String dmiValue) {
         if (!BooleanUtils.toBoolean(dmiValue)) {
-            LOG.debug("DMI is disabled, adding DMI related excluded patterns");
+            if (LOG.isDebugEnabled()) {
```

Comment:
Please revise all `if (LOG...` statements in this PR according to [this answer](https://stackoverflow.com/a/29561054/1362623).